jvb/src/main/java/org/jitsi/videobridge/rest/root/debug/Debug.java

```diff
@@ -161,6 +161,35 @@ public Response setEndpointFeatureState(
         return Response.ok().build();
     }
 
+    @GET
+    @Path("/features/endpoint/{confId}/{epId}/{feature}/")
+    public Boolean getEndpointFeatureState(
+        @PathParam("confId") String confId,
+        @PathParam("epId") String epId,
+        @PathParam("feature") EndpointDebugFeatures feature)
+    {
+        Conference conference = videobridge.getConference(confId);
+        if (conference == null)
+        {
+            throw new NotFoundException("No conference was found with the specified id.");
+        }
+
+        AbstractEndpoint endpoint = conference.getEndpoint(epId);
```

Comment:
You can use `getLocalEndpoint()` and avoid `isFeatureEnabled` being part of the `AbstractEndpoint` API

Reply:
Think I should do this for `setFeature` as well?